src/iam/iam-base.ts

```diff
@@ -67,6 +70,14 @@ export type Transaction = {
   from: string;
 };
 
+export type ChainConfig = {
```

Comment:
Maybe consider changing this to an interface so that it shows in TypeDoc documentation. I don't think types get picked up, but interfaces do?